### DISCUSSION TITLE
Nvidia fan control

### DISCUSSION THREAD

**Discussion by Blade7658** [Rem0o/FanControl.Releases#1372] [Q&A]
I know this will be frustrating question, but I have done what I can think of (including searching here and doing the things I found on this site). Love your program BTW, was very happy to support you (in a small way I am sure, but hopefully a lot of folks are chipping in a small amount).

For the first few months Fan Control was allowing my ROG Strix 3080 10GB center fan to go to zero (at idle) but lately something is taking it over (I get the ? mark) and even on manual it will not drop below 1000 RPM. I see online this is "a known problem" but I can't find a solution. Can you point me in the right direction? Thanks!

BTW, I do wish I could control my AIO fans, but I do that via iCUE, so it all comes together for a nice experience.

**Comment by Rem0o** [comment]
Does it get stuck at 30%?  That's a known limitation.  Make sure to check the following: 
![image](https://user-images.githubusercontent.com/14118956/207480175-b45aa51d-472d-430d-b457-940b5a933eca.png)

**Reply by Blade7658** [reply]
Thanks, that is checked, but as I read the explanation with it, I realized I had allowed my computer to run a bit hotter for much quieter. At idle my GPU runs at 45 (which I felt like was not too warm). Maybe that is above the 0 RPM limit, I will play around a bit with it. Thanks, helpful!

**Reply by Rem0o** [reply]
Yeah you must figure out at what temp your GPu normally goes 0 rpm, and tweak your curves around that.

**Reply by Blade7658** [reply]
Nope, that wasn't it. I turned all the fans (including case and AIO) to 100%. The GPU and CPU were 32 (that seems very cool to me) and the GPU fans (there are 3 but two channels) still insisted on 50%/1000 RPM. If you have any other ideas, but I will also keep researching. I know it used to go to zero, so I am thinking it was some update NVIDIA did.

**Reply by Tw33die** [reply]
Hey, I think I have the same problem maybe. The GPU fans will always default to 30% speed (around 1040rpm) no matter the temps. Even when the GPU temps are 20 degrees. 

I have a graph set up so that FC should only turn the fans on when the card temps exceed 30 degrees. 

If I toggle control off then on again, it drops to 0% below 30 degrees. However, whenever it then hits 30 degrees, the fans will come on (good) but STAY ON (bad) even when temps drop back below that threshold. 

I dunno.

**Reply by Skydiver777** [reply]
Hey there, just came across the same Problem. My GPU Fans (3080 ti) won't go back to idle once they passed 30%. I have the correct setting for curve override, but still no luck. Only works with MSI Afterburner, not sure How they override this. 
Any Workaround known?

**Reply by kklittich** [reply]
I have an MSI GTX 1660 Super and if the fans got to zero it has a crazy ramping up and down that is super annoying (this was before FanControl - the onboard firmware must just suck). My solution was to set a minimum speed at 35% so it never turns off and FanControl saved the day. By the way thanks so much for the developers of this project, I absolutely love the relative simplicity to use the software combined with being able to tweak the hell out of it if you so desire. I have a ‘Moderate’ profile that is nearly silent most of the time - amazing. I dropped them a tip via PayPal and suggest you all do the same if you agree.

**Comment by domessina** [comment]
Same problem Asus Strix 2080ti. Nvidia setting with 30% like mentionned in the tooltip does not help. Asus Strix 570 Motherboard
